spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+         Reserved_2 : Reserved_16;
+         MAC_Cap : Boolean;
+         Encrypt_Cap : Boolean;
+         Unused_Meas_Fresh_Cap : Reserved_1;
+         Unused_Meas_Cap : Reserved_2;
+         Chal_Cap : Boolean;
+         Cert_Cap : Boolean;
+         Reserved_3 : Reserved_1;
+         Handshake_In_The_Clear_Cap : Boolean;
+         Key_Upd_Cap : Boolean;
+         H_Beat_Cap : Boolean;
+         Encap_Cap : Boolean;
+         PSK_Cap : PSK_Request_Cap;
+         Key_Ex_Cap : Boolean;
+         Mut_Auth_Cap : Boolean;
+         Pub_Key_ID_Cap : Boolean;
+         Reserved_4 : Reserved_15
+            then null
+               if --  Encrypt_Cap:
+                  --  If set, one or more of PSK_CAP or KEY_EX_CAP fields shall be specified
+                  --  accordingly to indicate support.
+                  (Encrypt_Cap = False or PSK_Cap = PSK_Req_Supported or Key_Ex_Cap = True)
+                  --  MAC_Cap:
+                  --  If set, Requester supports message authentication. If set, one or more of
+                  --  PSK_CAP or KEY_EX_CAP fields shall be specified accordingly to indicate
+                  --  support.
+                  and (MAC_Cap = False or PSK_Cap = PSK_Req_Supported or Key_Ex_Cap = True)
+                  --  Key_Ex_Cap:
+                  --  If set, Requester supports KEY_EXCHANGE messages. If set, one or more of
+                  --  ENCRYPT_CAP and MAC_CAP shall be set.
+                  and (Key_Ex_Cap = False or (Encrypt_Cap = True or MAC_Cap = True))
+                  --  If supported, one or more of ENCRYPT_CAP and MAC_CAP shall be set.
+                  and (PSK_Cap /= PSK_Req_Supported or (Encrypt_Cap = True or MAC_Cap = True))
+                  --  ENCAP_CAP:
+                  --  If mutual authentication is supported, this field shall be set.
+                  and (Mut_Auth_Cap = False or Encap_Cap = True)
+                  --  HANDSHAKE_IN_THE_CLEAR_CAP
+                  --  If set, KEY_EX_CAP shall also be set.
+                  and (Handshake_In_The_Clear_Cap = False or Key_Ex_Cap = True)
+                  --  HANDSHAKE_IN_THE_CLEAR_CAP
+                  --  If the Requester does not support encryption and message authentication, then
+                  --  this bit shall be zero.
+                  and (Encrypt_Cap = True or Mac_Cap = True or Handshake_In_The_Clear_Cap = False);
+      end message;
+
+   type Meas_Cap is
+      (Meas_Supported => 0,
+       Meas_Plain     => 1,
+       Meas_Signed    => 2)
+   with Size => 2;
+
+   type PSK_Resp_Cap is
+      (PSK_Resp_Unsupported     => 0,
+       PSK_Resp_Without_Context => 1,
+       PSK_Resp_With_Context    => 2)
+   with Size => 2;
+
+   --  ISSUE: Componolit/RecordFlux#609
+   --  This message differs between SPDM version 1.0 and 1.1.
+   --  Since the minor version is not accessible from
+   --  within this message one message is needed for each version
+   --  that is selected by a refinement.
```

Comment:
Componolit/RecordFlux#609 is fixed. This actually is part of #92 where the duplicate message should be merged again.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.